src/main/java/de/hysky/skyblocker/skyblock/shortcut/ShortcutsConfigScreen.java

```diff
@@ -36,15 +37,21 @@ public void setTooltip(Text tooltip) {
     @Override
     protected void init() {
         super.init();
-        shortcutsConfigListWidget = new ShortcutsConfigListWidget(client, this, width, height - 96, 32, 25);
+        if (initialized) {
+            shortcutsConfigListWidget.setDimensions(width, height - 96);
+            shortcutsConfigListWidget.updatePositions();
+        } else {
+            shortcutsConfigListWidget = new ShortcutsConfigListWidget(client, this, width, height - 96, 32, 25);
+            initialized = true;
+        }
         addDrawableChild(shortcutsConfigListWidget);
         GridWidget gridWidget = new GridWidget();
         gridWidget.getMainPositioner().marginX(5).marginY(2);
         GridWidget.Adder adder = gridWidget.createAdder(2);
         buttonDelete = ButtonWidget.builder(Text.translatable("selectServer.delete"), button -> {
             if (client != null && shortcutsConfigListWidget.getSelectedOrNull() instanceof ShortcutsConfigListWidget.ShortcutEntry shortcutEntry) {
                 scrollAmount = shortcutsConfigListWidget.getScrollAmount();
-                client.setScreen(new ConfirmScreen(this::deleteEntry, Text.translatable("skyblocker.shortcuts.deleteQuestion"), Text.translatable("skyblocker.shortcuts.deleteWarning", shortcutEntry), Text.translatable("selectServer.deleteButton"), ScreenTexts.CANCEL));
+                client.setScreen(new ConfirmScreen(this::deleteEntry, Text.translatable("skyblocker.shortcuts.deleteQuestion"), Text.stringifiedTranslatable("skyblocker.shortcuts.deleteWarning", shortcutEntry), Text.translatable("selectServer.deleteButton"), ScreenTexts.CANCEL));
```

Comment:
I don't see the use of this change

Reply:
We have to use `stringifiedTranslatable` for non string, text, or primitive args.